prime-router/src/main/kotlin/DocumentationFactory.kt

```diff
@@ -26,11 +27,13 @@ object DocumentationFactory {
         val csvField = if (element.csvFields?.isNotEmpty() == true) element.csvFields[0] else null
         val sb = StringBuilder()
         val displayName = csvField?.name ?: element.name
+        val internalName = element.name
```

Comment:
Suggestion: This variable is only used one.  Maybe just inline it in line 36.

Reply:
Done. Thanks